.github/workflows/main.yml

```diff
@@ -24,6 +26,7 @@ jobs:
         run: |
           python -m pip install --upgrade setuptools pip
           python -m pip install -e .[tests]
+          python -m pip install -r requirements.txt
```

Comment:
So in the preceding line we're installing the test dependencies from the setup.py but then the next line we're switching back to a requirements file?

Reply:
I couldn't get the test dependencies to pick up the cryptography, and other required packages, for some reason.